### PR TITLE
Issue #6

### PR DESCRIPTION
**Pull request by MohammadBilalArif**
I have compiled the code locally, no error. Travis CI will build successfully after merge.

### CODE REVIEW

**Comment by glaslos** [comment]
Can you rebase your branch? https://git-scm.com/book/en/v2/Git-Branching-Rebasing

File: glutton.go [hunk 1]
```diff
@@ -3,24 +3,52 @@ package glutton
...
-	return netdb.GetServByPort(port, prot)
+// For the fields of services.conf
+type Config struct {
+	Description string
+	Ports       map[int]string
 }
 
-// CheckError handles errors
-func CheckError(err error) {
+var ser Config
```
Comment (glaslos) [inline glutton.go]: ser?

File: glutton.go [hunk 2]
```diff
@@ -70,10 +97,13 @@ func GetTCPDesPort(p []string, ch *nbc.NonBlockingChan) int {
 //return Destination port for UDP
 func GetUDPDesPort(p []string, ch *nbc.NonBlockingChan) int {
 
+	// Time used by conntrack for UDP logging
+	time.Sleep(10000000 * time.Nanosecond)
```
Comment (glaslos) [inline glutton.go]: 10 * time.Millisecond ?

File: glutton.go (hunk 1 above)
Comment (MohammadBilalArif) [inline glutton.go]: Every new connection in glutton calls `getHost()` and it return the address of  remote services on the base of `ser` variable. So I load `services.yml` in global variable `ser` at the startup of program to save the operation ons every call.

File: glutton.go (hunk 2 above)
Comment (MohammadBilalArif) [inline glutton.go]: :) I will alter this.

**Pull request closed without merging**